rootfs/usr/share/libalpm/hooks/package-cleanup.hook

```diff
@@ -8,4 +8,4 @@ Target = *
 Description = Cleaning up package cache...
 Depends = coreutils
```

Comment:
Changing /usr/bin/rm to /usr/bin/find means that this now depends on findutils instead of coreutils.

Disregarding, of course, that both packages should be installed on any POSIX-compliant Arch system. :/